kythe/proto/serving.proto

```diff
@@ -144,6 +147,10 @@ message File {
   string encoding = 3;
 
   FileInfo info = 4;
+
+  // Whether the File did not have a text fact.  Necessary to differentiate
+  // between an empty text and missing text.
```

Comment:
proto3 supports `optional` now which can differentiate between missing and empty; is that a viable option instead?

Reply:
That works

Reply:
Ah, I see proto3 is working its way back toward being proto2 😂